scripts/setup_uitests.sh

```diff
@@ -0,0 +1,12 @@
+# Setup database for uitests
+set -x
+
+git clone --depth 1 https://github.com/mozilla/normandy.git
```

Comment:
we should clone this to `~/normandy` to avoid any issues.

the paths below will need to be updated as well.

Reply:
What do you mean? Git clone will make a folder named normandy

Reply:
something like `git clone https://github.com/mozilla/normandy ~/normandy`

also the `--depth` option is not necessary.

Reply:
after the above you'd need to update the lines below to something like:
```
docker-compose -f ~/normandy/ci/docker-compose.yml up -d --build
docker-compose -f ~/normandy/ci/docker-compose.yml ps
sleep 10
docker-compose -f ~/normandy/ci/docker-compose.yml up -d
docker-compose -f ~/normandy/ci/docker-compose.yml exec web python manage.py createsuperuser --noinput --username user --email user@example.com
docker-compose -f ~/normandy/ci/docker-compose.yml exec web python manage.py migrate
docker-compose -f ~/normandy/ci/docker-compose.yml exec web python manage.py update_product_details
docker-compose -f ~/normandy/ci/docker-compose.yml exec web python manage.py initial_data
```

Reply:
ah, okay. What issues would this solve?

Reply:
i think it's just good practice not to clone a git repo within another git repo